test/pipelines/test_eval_batch.py

```diff
@@ -607,9 +608,24 @@ def test_extractive_qa_eval_simulated_top_k_reader_and_retriever(reader, retriev
 @pytest.mark.parametrize("document_store_with_docs", ["memory"], indirect=True)
 @pytest.mark.parametrize("reader", ["farm"], indirect=True)
 def test_extractive_qa_eval_isolated(reader, retriever_with_docs):
+    labels = deepcopy(EVAL_LABELS)
+    labels[0].labels.append(
```

Comment:
Looks good to me. It's a bit hidden to somebody reading the code though that the ids of two labels are the same and the answer spans are different. 
One option is to add `assert labels[0].labels[0].id == labels[0].labels[1].id`.
I thought about an alternative, which would be to explicitly copy the label `labels[0].labels[0]`, change only the answer in that label, and append that changed copy to `labels[0].labels`. That way, we ensure that two labels have the same id even if `EVAL_LABELS` is changed globally for some reason in future. However, that alternative is not really better.

Reply:
There is the same alternative in `test_extractive_qa_eval_isolated`. For the review I manually checked that the document id is `a0747b83aea0b60c4b114b15476dd32d ` in the two test cases and in `EVAL_LABELS `. 👍

Reply:
Thanks, those are some good points. I update the tests accordingly.